core/src/main/scala/org/http4s/EntityEncoder.scala

```diff
@@ -148,23 +146,23 @@ trait EntityEncoderInstances extends EntityEncoderInstances0 {
 
   // TODO parameterize chunk size
   // TODO if Header moves to Entity, can add a Content-Disposition with the filename
-  implicit def fileEncoder[F[_]](implicit F: Suspendable[F]): EntityEncoder[F, File] =
-    inputStreamEncoder[F].contramap(file => F.delay(new FileInputStream(file)))
+  implicit def fileEncoder[F[_]](implicit F: Sync[F]): EntityEncoder[F, File] =
+    inputStreamEncoder[F, FileInputStream].contramap(file => F.delay(new FileInputStream(file)))
 
   // TODO parameterize chunk size
   // TODO if Header moves to Entity, can add a Content-Disposition with the filename
-  implicit def filePathEncoder[F[_]: Suspendable]: EntityEncoder[F, Path] =
+  implicit def filePathEncoder[F[_]: Sync]: EntityEncoder[F, Path] =
     fileEncoder[F].contramap(_.toFile)
 
   // TODO parameterize chunk size
-  implicit def inputStreamEncoder[F[_]: Suspendable]: EntityEncoder[F, F[InputStream]] =
-    streamEncoder[F, Byte].contramap { in: F[InputStream] =>
-      readInputStream[F](in, DefaultChunkSize)
+  implicit def inputStreamEncoder[F[_]: Sync, IS <: InputStream]: EntityEncoder[F, F[IS]] =
```

Comment:
I did this to make the tests work, without adding more specific types at the call site when called with a subclass of `InputStream`.

I have no clue if there would be any issues with this, and I don't understand why it's suddenly needed when it worked before parameterising.